### PR TITLE
added documentation for delete self endpoint of doord service

### PR DESCRIPTION
**Pull request by valioprea**
I misspelled the branch name :/

### CODE REVIEW

File: source/includes/_account.md [hunk 1]
```diff
@@ -499,3 +499,24 @@ This endpoint updates a user's details, currently this is limited to display nam
...
+## Delete Account
+
+Deletes the currently logged in user's account, associated with the application the user is logged in.
+<br>
+<b>Warning! </b>This operation is irreversible!
+
+```shell
+curl 'https://api.doordeck.com/account' \
+   -X DELETE \
+  --header 'Content-Type: application/json' \
```
Comment (mpbarnwell) [inline source/includes/_account.md:512]:
```suggestion
```

Don't think you need a content type

**Comment by valioprea** [comment]
Will close this: https://github.com/doordeck/doord/issues/63

File: source/includes/_account.md [hunk 2]
```diff
@@ -499,3 +499,24 @@ This endpoint updates a user's details, currently this is limited to display nam
 Parameter | Required | Description
 --------- | ------- | -----------
 displayName | true | User's desired display name
+
+## Delete Account
+
+Deletes the currently logged in user's account, associated with the application the user is logged in.
+<br>
+<b>Warning! </b>This operation is irreversible!
```
Comment (mpbarnwell) [inline source/includes/_account.md:507]: We tend to use the warning/aside system, it makes a big red box:
```suggestion
<aside class="warning">
This operation is irreversible!
</aside>
```

File: source/includes/_account.md [hunk 3]
```diff
@@ -499,3 +499,24 @@ This endpoint updates a user's details, currently this is limited to display nam
...
+  --header 'Authorization: Bearer TOKEN' \
+```
+
+### HTTP Request
+
+`DELETE https://api.doordeck.com/account`
+
+### Request Parameters
+
+This request has no body/ parameters. The applicationId and userId are taken from the authentication token.
```
Comment (mpbarnwell) [inline source/includes/_account.md:522]:
```suggestion
```

I think we can just get rid of this section.

**Review by mpbarnwell — CHANGES_REQUESTED** [review]

**Review by mpbarnwell — APPROVED** [review]

**Pull request merged**